cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/store/DefaultStoreTest.java

```diff
@@ -663,4 +669,70 @@ public void testCheckDeletedProceduresAndWorkflow () throws Exception {
     Assert.assertEquals(0, specsToBeDeleted.size());
   }
 
+  @Test
+  public void testAdapterMDSOperations() throws Exception {
+    Id.Namespace namespaceId = new Id.Namespace("testAdapterMDS");
+
+    Map<String, String> properties = ImmutableMap.of("frequency", "10m");
+    List<Source> sources = Lists.newArrayList(new Source("eventStream", DataType.STREAM,
+                                                         ImmutableMap.of("prop1", "val1")));
+
+    List<Sink> sinks = Lists.newArrayList(new Sink("myAvroFiles", DataType.DATASET,
+                                                   ImmutableMap.of("type", "co.cask.cdap.data.dataset.Fileset")));
+
+    AdapterSpecification specStreamToAvro = new AdapterSpecification("streamToAvro", AdapterType.BATCH,
```

Comment:
really - `streamToAvro` shouldn't be the name of the adapter instance, right? That should be the adapterType.